src/verify_cert.rs

```diff
@@ -652,6 +664,9 @@ where
     for v in values {
         match f(v) {
             Ok(()) => return Ok(()),
+            Err(Error::MaximumSignatureChecksExceeded) => {
```

Comment:
could do `Err(Error::MaximumSignatureChecksExceeded) @ err => { return err },` here to avoid writing it twice